source/deployment/resyncing-conda.rst

```diff
@@ -8,29 +8,35 @@ operating cycle, to help isolate users from changes that might break things
 mid-cycle due to some unintended upgrade.
 
 Before the start of each cycle, we re-synchronize our internal collection of
-packages with the latest packages available from Anaconda.
+packages with the latest packages available from Anaconda (defaults channel).
```

Comment:
I have a minor recommendation to use double-backticks or quotes surrounding `defaults` to make it distinguishable.